contrib/terraform/vsphere/variables.tf

```diff
@@ -37,6 +35,13 @@ variable "ssh_public_keys" {
 variable "template_name" {}
 
 # Optional variables (ones where reasonable defaults exist)
+variable "vapp" {
+  default = false
+}
+
+variable "interface_name" {
+  default = "ens192"
```

Comment:
I've not kept up to date with vsphere evolutions lately but why is this a default instead of `eth0`?

Reply:
ens### is a new notation now that udev names the interface which should make the naming more predictable. It is the default in many distro's and is not related to vSphere specific deployments. 192 seems to be de default number for a ubunut 20.04 cloud image on vSphere. This default was also hardcoded in vapp-cloud-init.tpl before.